src/generic/tests/test_decorators.py

```
@@ -0,0 +1,34 @@
+# Copyright (C) 2017 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+"""
+OVS-extensionds decorator test class
+"""
+
+import time
+import unittest
+from ovs_extensions.generic.decorators import timeout, TimeoutError
+
+
+class TestDecorators(unittest.TestCase):
+
+    def test_timeout(self):
```

Only proves that the error would raise

There should be a test asserting that the method is actually aborted (or not, I do not know what the nature of the decorator is).
This can be tested by writing to a file, write something every second, let it raise, wait for all seconds to pass, check file contents